crates/cxx-qt-gen/src/parser/invokable.rs

```diff
@@ -28,3 +33,55 @@ pub struct ParsedQInvokable {
     /// Any specifiers that declared on the invokable
     pub specifiers: HashSet<ParsedQInvokableSpecifiers>,
 }
+
+impl ParsedQInvokable {
```

Comment:
this part has moved from `parse_impl_items` ? maybe split this into a separate commit so we can land this separately while the rest is being prepared, then this also reduces the noise in this commit.

Reply:
Right, ideally I'd want to split a few of the refactorings I did into it's own PR. Will see if I can find the time before my QML training next week :thinking:

Reply:
Done